AppleStore/ViewControllers/BuyViewController.swift

```diff
@@ -8,8 +8,18 @@
 import UIKit
 /// BuyViewController
 final class BuyViewController: UIViewController {
+    
+    var delegate: SwitchModesDelegate?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .black
+        
+        view.backgroundColor = UIColor(named: "viewBackground")
```

Comment:
```suggestion
        view.backgroundColor = UIColor(named: "")
```
В константы, цвет через Asset